src/matrix/ssss/recoveryKey.ts

```diff
@@ -13,17 +13,24 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-import {Key} from "./common.js";
+import {Key} from "./common";
+import {KeyDescription} from "./common";
+import type {Platform} from "../../platform/web/Platform.js";
 
-const OLM_RECOVERY_KEY_PREFIX = [0x8B, 0x01];
+export type Olm = {
```

Comment:
Hmm, [libolm has types](https://gitlab.matrix.org/matrix-org/olm/-/blob/master/javascript/index.d.ts), can we do `import type * as Olm from "@matrix-org/olm"`?

Reply:
Can't get this to work: `Cannot use namespace Olm as type`
 But I'm convinced that I'm missing something because the type declaration for `aes-js` also did something similar :thinking:

Reply:
Maybe I need to do `typeof Olm` as opposed to `Olm`